### ISSUE TITLE
Implement 2-layer NN in TensorFlow for MNIST

### ISSUE THREAD

**Issue by anton-dergunov** [anton-dergunov/ml-playground#11]
- Using TF.Learn
- Then using lower-level API (relu, softmax, initialize weights with truncated normal distribution), and compare
